src/network/tarcap/threadpool/packets_queue.cpp

```diff
@@ -0,0 +1,48 @@
+#include "packets_queue.hpp"
+
+namespace taraxa::network::tarcap {
+
+PacketsQueue::PacketsQueue(size_t max_workers_count)
+    : packets_(), MAX_WORKERS_COUNT(max_workers_count), act_workers_count_(0) {}
```

Comment:
MAX_WORKERS_COUNT -> naming?

Reply:
I dont get this comment

Reply:
it is class variable, not sure if we should use capital letters